### ISSUE TITLE
Ability to open just a folder of strings files

### ISSUE THREAD

**Issue by BrianSemiglia** [mohakapt/Stringz#21]
It'd be nice if Stringz could generate a Stringz specific file from an xcode project that could be sent off to a 3rd party for translations and then on return of that edited file could be applied back to the xcode project.

**Comment by mohakapt** [comment]
Yeah, that would be a nice feature. Although there is a similar thing already included in Xcode, It allows you to export .string files to send time to your translator but it doesn't provide a convenient way for the translator to work on them. I'll definitely consider implementing such a feature.

Right now my priority is to integrate this [library](https://github.com/JiriTrecak/Laurine) into Stringz, I've been using it for a while now and it's a fantastic library to make strings auto-completable.
